utils/vscode/snippets/carbon.json

```diff
@@ -0,0 +1,89 @@
+{
+    "Create a Function": {
+        "prefix": "function",
+        "body": [
+            "fn ${1:name} {",
+            "   return $2;",
+            "}"
+        ],
+        "description": "Carbon Function: A function is a block of code that can be called from other code. Functions can be used to create custom logic, to handle events, or to create reusable components."
+    },
+    "Class": {
+        "prefix": "class",
+        "body": [
+            "class ${1:name} {",
+            "   var $2;",
+            "};"
+        ],
+        "description": "Carbon Class: A class is a block of code that can be used to create objects. Classes can be used to create reusable components or to serve as a prototype for other objects."
+    },
+    "If Statement": {
+        "prefix": "if",
+        "body": [
+            "if (${1:condition}) {",
+            "   ${2:code}",
+            "}"
+        ],
+        "description": "Carbon If: An if statement is a block of code that is executed if a condition is true."
+    },
+    "Else Statement": {
+        "prefix": "else",
+        "body": [
+            "else {",
+            "   ${1:code}",
+            "}"
+        ],
+        "description": "Carbon Else: An else statement is a block of code that is executed if a condition is false."
+    },
+    "If Else Statement": {
+        "prefix": "ifelse",
+        "body": [
+            "if (${1:condition}) {",
+            "   //${2:code}",
+            "} else {",
+            "   //${3:code}",
+            "}"
+        ],
+        "description": "Carbon If: An if statement is a block of code that is executed if a condition is true. And else block of code is executed when condition is false."
+    },
+    "For Loop": {
+        "prefix": "for",
+        "body": [
+            "for (int $1 = 0; $1 < $2; $1++) {",
+            "\t$3",
```

Comment:
why is using `\t` when other's are using spaces? I think there is a special token for tab in snippets.